src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,17 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+export const TodoInfo = ({ todo }) => {
+  const { completed, title, user } = todo;
+
+  return (
+    <article
+      className={completed ? 'TodoInfo TodoInfo--completed'
+        : 'TodoInfo'}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+
+      <UserInfo user={user} />
```

Comment:
Make sure the 'UserInfo' component properly handles the case where 'user' is null or undefined.
So, just add a condition